update fuji docs in tooling section (#1963)
ava-labs/avalanche-docs: content/docs/tooling/create-deploy-avalanche-l1s/deploy-on-fuji-testnet.mdx

| VM ID              | qcvkEX1zWSz7PtGd7CKvPRBqLVTzA7qyMPvkh5NMDWkuhrcCu  |
+--------------------+----------------------------------------------------+
| Blockchain ID      | 2U7vNdB78xTiN6QtZ9aetfKoGtQhfeEPQG6QZC8bpq8QMf4cDx |
+--------------------+                                                    +
| P-Chain TXID       |                                                    |
+--------------------+----------------------------------------------------+
```

To get your new Avalanche L1 information, visit the [Avalanche L1 Explorer](https://subnets-test.avax.network/). The search best works by blockchain ID, so in this example, enter `2U7vNdB78xTiN6QtZ9aetfKoGtQhfeEPQG6QZC8bpq8QMf4cDx` into the search box and you should see your shiny new blockchain information.

Add a Validator[​](#add-a-validator "Direct link to heading")
-------------------------------------------------------------

Before proceeding to add a validator to our Avalanche L1, we will need to have the validator's NodeID, BLS public key and proof of possession. These can be obtained by ssh into the node itself and run the `getNodeID` API specified [here](https://docs.avax.network/api-reference/info-api#infogetnodeid)

> To add a validator to an Avalanche L1, the owner of the key that acts as the controller of ValidatorManager contract specified in `avalanche blockchain create` command above run:

Reviewer: ```suggestion
To add a validator to an Avalanche L1, the owner of the key that acts as the controller of `ValidatorManager` contract specified in `avalanche blockchain create` command above run:
```

```bash
Use the arrow keys to navigate: ↓ ↑ → ←
?  Which key should be used to pay for transaction fees on P-Chain?:
    test
  ▸ mytestkey
```

Now enter the **NodeID** of the new validator to be added.

```bash
What is the NodeID of the validator you'd like to whitelist?: NodeID-BFa1paAAAAAAAAAAAAAAAAAAAAQGjPhUy
```

Next, enter the node's BLS public key and proof of possession.

Now, enter the amount of AVAX that you would like to allocate to the new validator.

The validator's balance is used to pay for continuous fee to the P-Chain. When this Balance reaches 0, the validator will be considered inactive and will no longer participate in validating the L1.

1 AVAX should last the validator about a month.

```bash
What balance would you like to assign to the validator (in AVAX)?: 1
```

Next, select a key that will receive the leftover AVAX if the validator is removed from the L1:

```bash
 Which stored key should be used be set as a change owner for leftover AVAX?:
    test
  ▸ mytestkey
```

Next, select a key that can remove the validator:

```bash
? Which stored key should be used be able to disable the validator using P-Chain transactions?:
    test
  ▸ mytestkey
```

> By the end of the command, you would have successfully added a new validator on the given Avalanche L1 on `Fuji`!

Reviewer: ```suggestion
By the end of the command, you would have successfully added a new validator to your Avalanche L1 on the Fuji Testnet!
```
